lib/thrift/generator/utils.ex

```diff
@@ -60,30 +60,47 @@ defmodule Thrift.Generator.Utils do
   end
 
   @doc """
-  Merge the binaries in an iolist.
+  Optimize an expression that returns an iolist.
 
-    ["a", "b", ["c", [var]]] => ["abc", var]
+    [<<1>>, <<2>>] => <<1, 2>>
```

Comment:
That's a good thought. It's tricky to represent nicely as runnable code though, since it works on quoted expressions. That's why UtilsTest uses a macro.

Really, these functions are not something I would expect people to try to re-use. Or I hope that don't. Maybe we should `@moduledoc false` this whole thing.